superbench/benchmarks/micro_benchmarks/cudnn_function.py

```diff
@@ -0,0 +1,441 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the cudnn functions benchmarks."""
+
+import os
+import json
+import yaml
+
+from superbench.common.utils import logger
+from superbench.benchmarks import Platform, BenchmarkRegistry
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class CudnnBenchmark(MicroBenchmarkWithInvoke):
+    """The cudnn performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self.__default_params_dict_list = [
+            {
+                'name': 'cudnnConvolutionBackwardFilter',
+                'algo': 0,
+                'input_dims': [32, 128, 14, 14],
+                'input_stride': [25088, 196, 14, 1],
+                'input_type': 0,
+                'output_dims': [32, 32, 14, 14],
+                'output_stride': [6272, 196, 14, 1],
+                'conv_type': 0,
+                'use_tensor_core': False,
+                'arrayLength': 2,
```

Comment:
naming is inconsistent. arrayLength => array_length?
Please check all the variables.